multi-node/aws/pkg/config/templates/cloud-config-controller

```diff
@@ -31,7 +34,9 @@ coreos:
             [Service]
             ExecStartPre=/usr/bin/curl --silent -X PUT -d \
             "value={\"Network\" : \"{{.PodCIDR}}\", \"Backend\" : {\"Type\" : \"vxlan\"}}" \
-            http://localhost:2379/v2/keys/coreos.com/network/config?prevExist=false
+            {{if not .DedicatedEtcd}}http://localhost:2379/v2/keys/coreos.com/network/config?prevExist=false
+            {{else}}http://{{index .EtcdIPs 0}}:2379/v2/keys/coreos.com/network/config?prevExist=false
```

Comment:
Good point. We should probably use etcdctl instead of curl and have it use all configured etcd endpoints.